### PR TITLE
EventHub: moved VNet, IpFilter rules and Kafka from 2018-preview to 2017 API Version

### PR DESCRIPTION
**Pull request by v-Ajnava**
approved PR  for 2018-preview API Version : https://github.com/Azure/azure-rest-api-specs/pull/3294

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by dsgouda** [comment]
@v-Ajnava could you create a commit of original files copied into this version before changes from the new version were introduced

**Comment by v-Ajnava** [comment]
@dsgouda , we are moving the VNetRule, IPFilterRule and EnablleKafka from 2018-preview to 2017 API version. For 2018-Preview version the only .Net SDK was published with Version 2.8.0-preview. When publishing the new SDK for 2017 (stable) the current PR, we will bump the Major version which will be - 3.0.0

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part, please ensure `PUT`/`DELETE` operations are not long running

File: specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json [hunk 1]
```diff
@@ -1856,6 +1856,376 @@
...
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```
[the remark this answers is not in the document]
Comment (v-Ajnava) [reply specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json]: @dsgouda , PUT and DELETE for namespaces are long running operations, do we need to change that ?

[the remark this answers is not in the document]
Comment (dsgouda) [reply specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json]: I guess this operation is for Namespace IPFilterRule, please check the same

[the remark this answers is not in the document]
Comment (v-Ajnava) [reply specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json]: @dsgouda ,I checked the swagger, only namespace PUT and DELETE are with  "x-ms-long-running-operation": true.
The Namespace IPFilterRule dose not have '"x-ms-long-running-operation": true.'

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**